### ISSUE TITLE
Nibre - please release the source so we can help you :)

### ISSUE THREAD

**Issue by er453r** [Nibre/MotherVR#231]
@Nibre the strength of open-source community is that we can help each other to make the projects grow :) 

Please release the source (as it is, we do not judge!) and we can take it from there.

Yours truly,
Martin

**Comment by spyro2000** [comment]
I wonder why he didn't react at all :(

**Comment by ayuhito** [comment]
It would also be great if you keep in mind to find another person to maintain this repository and merge PRs in your stead from the first batch of contributors. Setting up a structure with at least 2 people having write access would reduce the burden on you and help maintain the longevity of this project should one person go MIA.

**Comment by er453r** [comment]
yeah, just look at the success of projects like https://github.com/praydog/REFramework

**Comment by Ionwerks** [comment]
Yes, please! It's hard to imagine there's anything to gain from keeping it closed source now, just eventual irrelevance.

**Comment by delferius** [comment]
Please dude! Share the source code to maintain this amazing Mod!

**Comment by Void48** [comment]
+1 please!!

**Comment by javaskript** [comment]
I’ve messaged Nibre in various formats, emailed him, no response. It’s been years since he last visited this. The guy obviously has no intention of touching it again.

**Comment by spyro2000** [comment]
@javaskript ...which is completely fine, he already provided an awesome base! It's just sad, that he didn't share the code but maybe there are some copyright or legal concerns :(

**Comment by javaskript** [comment]
> @javaskript ...which is completely fine, he already provided an awesome base! It's just sad, that he didn't share the code but maybe there are some copyright or legal concerns :(

Of course, he can abandon the project if he wishes but it’s just unfortunate that he said he’d release the source shortly and never did 🤷‍♂️

**Comment by MulleredOG** [comment]
Nibre abandoned the project around the same time he was employed by 343 industries to work on Halo. I'd imagine he was told to due to conflict of interest etc. I doubt he will be coming back to this anytime soon as long as he is employed by them.  You cant blame him really. The only way this can progress if if someone with the knowledge comes along and writes a new mod. You have to remember, Nibre didn't create the VR mode, it was built into the game by the developers as a tech demo. Nibre basically found a way to unlock the mode on modern headsets shortly after oculus changed the way their headsets and software worked and the mode stopped working on dual screen headsets. It shouldn't be too hard for someone with the right knowledge to unpick it and develop it further. It may even be better if someone injects a new VR mode like Praydog is doing with unreal engine games, as a lot of the issues are inherent to original VR mode built into the game such as the dodgy off centre head movement etc
